coursier.bzl

```diff
@@ -466,16 +467,24 @@ def _deduplicate_artifacts(dep_tree):
 # Get the path to the cache directory containing Coursier-downloaded artifacts.
 #
 # This method is public for testing.
-def get_coursier_cache_or_default(env_dict, use_unsafe_shared_cache):
-    location = env_dict.get("COURSIER_CACHE")
-
-    # If we're not using the unsafe shared cache, or if COURSIER_CACHE is not defined,
-    # use 'external/<this repo>/v1/'. 'v1/' is the current version of the Coursier cache.
-    if not use_unsafe_shared_cache or location == None:
+def get_coursier_cache_or_default(os_env, os_name, use_unsafe_shared_cache):
+    # If we're not using the unsafe shared cache use 'external/<this repo>/v1/'.
+    # 'v1' is the current version of the Coursier cache.
+    if not use_unsafe_shared_cache:
         return "v1"
-    else:
+
+    coursier_cache_env_var = os_env.get("COURSIER_CACHE")
+    if coursier_cache_env_var:
         # This is an absolute path.
-        return location
+        return coursier_cache_env_var
+
+    # locations from https://get-coursier.io/docs/1.1.0-M14-1/cache.html#default-location
+    if _is_windows(os_name):
+        return "%s\\Coursier\\Cache\\v1" % os_env.get("LOCALAPPDATA")
```

Comment:
Sorry for the delay, I just managed to try this on a Windows machine. 

While pinning artifacts on Windows is still pretty broken with the `pin` script not working properly (https://github.com/bazelbuild/rules_jvm_external/issues/324), this should be `"%s/Coursier/cache/v1" % os_env.get("LOCALAPPDATA").replace("\\", "/")`, based on what I've tested.

Reply:
Thanks for testing it on Windows!  Is it supposed to be lower case `cache` in the path or captilized `Cache` ?

Reply:
It is `cache` on my machine.